### ISSUE TITLE
hyrolo-sort is broken

### ISSUE THREAD

**Issue by tekakutli** [rswgnu/hyperbole#168]
I think
* hello :)
so, Im using hyrolo to store useful commands, noticed the order broke with hyrolo-sort and investigated
sort works with plain lists, but with multi level ones doesnt
* to replicate:
I removed the commands leaving only the "headers", still un-works, so:
if you sort a .rolo.otl file with this list you will see guile being move up next to emacs, if you do it again you will see guile2 moved up too and so on
irc also jumps next to guix
```
*   emacs
**   agenda
**   dired
**   geiser
**   layout
**   magit
***   forge
***   history
***   rebase
**   multicursor
**   navigation
**   orgmode
***   tangle
*   guile
*   guile2
*   guix
**   guixshell
***   guixshellkid
*   irc
*   krita
*   linux
*   lisptips
*   regex
*   screenshot
*   sound
*   unicode
*   videos
**   ffmpeg
**   mpv
**   ytdlp
```
using master-branch from this repo
maybe I misunderstood how sort works
anyway, thank you for hyperbole

**Comment by rswgnu** [comment]
Please use {C-h h m c} to compose a message to the Hyperbole mailing list.  This will show us which version you are running.

But if you are using 8.0.0pre, from the git master, this is supposed to be fixed; was broken in 7.1.3.

-- Bob

**Comment by rswgnu** [comment]
Just FYI, this was fixed last week in the pre-release.
